index.html

```diff
@@ -0,0 +1,312 @@
+                ><i class="fab fa-twitter"></i
+              ></a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">English</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">My Page</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Logout</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="container-fluid second-nav">
+        <a class="navbar-brand hide-sm" href="home.html">
+          <span><i class="fas fa-archway"></i></span>
+          Historia Summit 2022</a
+        >
+        <button
+          class="navbar-toggler"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#navbarSupportedContent"
+          aria-controls="navbarSupportedContent"
+          aria-expanded="false"
+          aria-label="Toggle navigation"
+        >
+          <span class="navbar-toggler-icon">
+            <i class="menuIcon fas fa-bars"></i>
+            <i class="closeIcon fas fa-times"></i>
+          </span>
+        </button>
+        <div
+          class="collapse navbar-collapse justify-content-end"
+          id="navbarSupportedContent"
+        >
+          <ul class="navbar-nav mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a
+                class="nav-link active menuItem"
+                aria-current="page"
+                href="about.html"
+                >About</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active menuItem" aria-current="page" href="#"
+                >Program</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active menuItem" aria-current="page" href="#"
+                >Join</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active menuItem" aria-current="page" href="#"
+                >Sponsor</a
+              >
+            </li>
+            <li class="nav-item">
+              <a class="nav-link active menuItem" aria-current="page" href="#"
+                >News</a
+              >
+            </li>
+            <li class="nav-item campaign">
+              <a class="nav-link active menuItem" aria-current="page" href="#"
+                >Campaigns</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <!-- WEBSITE CONTENT  -->
+
+    <div class="container-fluid main-content px-0">
+      <section class="introduction">
+        <div class="container">
+          <div class="row justify-content-center">
+            <div class="col-md-9">
+              <h1 class="history intro-greeting">"Hello! Sharing World"</h1>
+              <h4 class="history intro-tag">
+                HISTORIA MAMBOLEO ART AND CULTURAL SUMMIT 2022
+              </h4>
```

Comment:
- [ ] To make your website look more attractive, I would recommend you use **cocogoose** font for the title as well as the summit date 😄